block/manager.go

```diff
@@ -177,6 +177,9 @@ func (m *Manager) Start(ctx context.Context) error {
 	}
 
 	/* ----------------------------- sequencer mode ----------------------------- */
+	// Subscribe to batch events, to update last submitted height in case batch confirmation was lost (e.g. on restart)
```

Comment:
```suggestion
	// Subscribe to batch events, to update last submitted height in case batch confirmation was lost. This could happen if the sequencer crash/restarted just after submitting a batch to the settelement and by the time we query the last batch, this batch wasn't accepted yet. 
```